spec/display_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'display'
+
+RSpec.describe Display do
+
+  before :each do
+    @display = Display.new
+  end
+
```

Comment:
Rspec has either nested `describe` blocks or `context` blocks. Which do nothing themselves, but can be used to group together tests on related methods.

In this case, it could be something like:
```
context 'displaying a grid' do
  it ' displays an empty board correctly' do
    // code
  end
end
```